test/integration/instance-controller-param-change/00-assert.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: kudo.k8s.io/v1alpha1
```

Comment:
Is there any reason not to split these tests out into three separate tests?

I think that would help the readability a lot.

Reply:
@gkleiman bumping this comment :)

Reply:
Thanks for bumping this, I had missed your comment =).